### PR TITLE
Store the answer in memory

### PR DESCRIPTION
**Pull request by Theweeksad**
Added JavaScript folders and scrip in HTML

### CODE REVIEW

File: Flow/question-one.html [hunk 1]
```diff
@@ -4,13 +4,58 @@
...
+                        id="answerThree"
+                        class="input__btn"
+                        type="radio"
+                        name="query1"
+                        value="3"
+                    />
+                    <label for="answerThree">Div</label>
+                </p>
+            </fieldset>
+            <input class="btn__next" type="Submit" value="Next">
```
Comment (ripsamuel) [inline Flow/question-one.html:54]: is type "submit" in lowercase

File: Javascript/Question-two.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const form = document.getElementById("form__list");
+
+form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  const elements = event.target.elements.namedItem('query2');
+
+  const isCorrect = elements[1].checked && elements[2].checked && !elements[0].checked;
```
Comment (ripsamuel) [inline Javascript/Question-two.js:7]: mustbe isCorrect is True if only the correct answer is selected

File: Flow/question-three.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                        min="1"
+                        max="3"
+                        step="1"
+                        name="query3"
+                        value="1"
+                    />
+                </p>
+
+                <p class="slider__item">
+                    <label for="Var">Var</label>
```
Comment (ripsamuel) [inline Flow/question-three.html:37]: dont repeat yourself

File: Flow/question-two.html [hunk 4]
```diff
@@ -4,13 +4,57 @@
...
+                    <label for="answerSix">The #ID does not exist</label>
+                </p>
+            </fieldset>
+            <input class="btn__next" type="submit" value="Next">
+        </form>
     </main>
+
+    <script src="../Javascript/Question-two.js"></script>
 </body>
 </html>
```
Comment (ripsamuel) [inline Flow/question-two.html:60]: pls add spaces

**Review by devdevsamuel — COMMENTED** [review]
anew pls attach screenshots to check apperance

File: Flow/question-one.html (hunk 1 above)
Comment (Theweeksad) [reply Flow/question-one.html:54]: hello, I have already fixed the submit

File: Flow/question-one.html [hunk 5]
```diff
@@ -4,13 +4,58 @@
...
+        </form>
+
     </main>
+
+    <script src="../Javascript/Question-one.js"></script>
 </body>
 </html>
```
[the remark this answers is not in the document]
Comment (Theweeksad) [reply Flow/question-one.html:61]: Hello, added new line at the end

File: Flow/question-two.html (hunk 4 above)
Comment (Theweeksad) [reply Flow/question-two.html:60]: Hello, added new line at the end

File: Flow/question-three.html (hunk 3 above)
Comment (Theweeksad) [reply Flow/question-three.html:37]: Hi, I fixed the don't repeat yourself, I changed the for attribute.

File: Javascript/Question-two.js (hunk 2 above)
Comment (Theweeksad) [reply Javascript/Question-two.js:7]: hello, if it were for one answer I would have to make the change you ask for, but it is valid that two answers are true and one is false.

**Pull request merged**